modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala

```diff
@@ -456,14 +469,20 @@ class SeqTranslate(site: Site, systems: Systems[IO], settings: TranslateSettings
                       gwsHeaders(GwsEpics.instance, sys), h)))
         } yield read
 
-      case StepType.FlatOrArc(inst)       =>
+      case StepType.FlatOrArc(inst) =>
           calcInstHeader(config, sys).map(h => Reader(ctx =>
             List(commonHeaders(TcsEpics.instance, config, flatOrArcTcsSubsystems(inst).toList, sys)(ctx), gcalHeader(GcalEpics.instance, sys), gwsHeaders(GwsEpics.instance, sys), h)))
 
-      case StepType.DarkOrBias(_)      =>
+      case StepType.DarkOrBias(_)   =>
           calcInstHeader(config, sys).map(h => Reader(ctx => List(commonHeaders(TcsEpics.instance, config, Nil, sys)(ctx), gwsHeaders(GwsEpics.instance, sys), h)))
-      case StepType.AlignAndCalib         => TrySeq(Reader(_ => Nil)) // No headers for A&C
-      case st                    => TrySeq.fail(Unexpected(s"Unsupported step type $st"))
+
+      case StepType.AlignAndCalib   => TrySeq(Reader(_ => Nil)) // No headers for A&C
+
+      case StepType.Gems(_)         =>
```

Comment:
Why do we need a new `StepType`? Is it equivalent to the `Altair` step type?

Reply:
It is not a new type, but it was ignored until now. I agree it could be merged with `StepType.Altair`, if the code that handles both types ends up being equivalent. But let's wait until this is running before making that decision.

Reply:
👍